src/python/pants/backend/go/util_rules/third_party_pkg_test.py

```diff
@@ -30,15 +31,23 @@
     ThirdPartyPkgAnalysisRequest,
 )
 from pants.build_graph.address import Address
-from pants.engine.fs import Digest, Snapshot
+from pants.engine.fs import Digest
 from pants.engine.internals.scheduler import ExecutionError
 from pants.engine.process import ProcessExecutionFailure
 from pants.engine.rules import QueryRule
 from pants.testutil.rule_runner import RuleRunner, engine_error
+from pants.util.contextutil import temporary_dir
+
+
+@pytest.fixture(scope="module")
+def named_caches_dir() -> Iterator[str]:
```

Comment:
Maybe use append_only_caches naming here? Named caches is not the official name of the feature.

Reply:
it is the name of the command_line argument that this fixture controls, so I'm leaning to leave this as is.

Reply:
Sounds fine to me. Disregard my comment then.